diff --git a/node_modules/@hono/zod-openapi/dist/index.mjs b/node_modules/@hono/zod-openapi/dist/index.mjs
index 98a8a77..563704e 100644
--- a/node_modules/@hono/zod-openapi/dist/index.mjs
+++ b/node_modules/@hono/zod-openapi/dist/index.mjs
@@ -156,7 +156,10 @@ var OpenAPIHono = class _OpenAPIHono extends Hono {
     });
   };
   route(path, app) {
-    const pathForOpenAPI = path.replaceAll(/:([^\/]+)/g, "{$1}");
+
+    const pathForOpenAPI = typeof path === "string"
+    ? path.replace(/:([^\/]+)/g, "{$1}")
+    : String(path || "").replace(/:([^\/]+)/g, "{$1}");
     super.route(path, app);
     if (!(app instanceof _OpenAPIHono)) {
       return this;
